examples/tutorials/advanced/legends.py

```diff
@@ -0,0 +1,141 @@
+"""
+Creating legends
+================
+
+The :meth:`pygmt.Figure.legend` method creates legends, whereby auto-legends as well
+as manually created legends are supported.
+"""
+
+# %%
+import io
+
+import pygmt
+
+# %%
+# Create an auto-legend
+# ---------------------
+#
+# For auto-legends, the ``label`` parameter of :meth:`pygmt.Figure.plot` has to be
```

Comment:
Some other plotting methods (e.g., `plot3d`/`histogram`) also have the `label` parameter, so better to be general here and use `Figure.plot` as an example.

Reply:
I updated the explanation text in commit https://github.com/GenericMappingTools/pygmt/pull/3594/commits/49773b491774e44ca43e45ca8be89837c7d1dc0d.